src/transformers/integrations.py

```diff
@@ -284,26 +284,50 @@ def _set_if_auto(config, key, val):
 
 class DeepSpeedConfigHF:
```

Comment:
As this is now becoming a public class, should we think of a better name? I feel it's a bit too cryptic at the moment.

The key is to differentiate from DeepSpeed's own classes with similar name and to make it clear that this is a transformers/HF version.

Reply:
I would just use `HfDeepSpeedConfig` as the general rule used in other names is to put the HF first with the flowercase `HfAPI` (HfAPI, HfFolder, HfArgumentParser for instance)